common/src/View/UVView.cpp

```diff
@@ -125,6 +127,12 @@ namespace TrenchBroom {
             Refresh();
         }
         
+        void UVView::documentWasCleared(MapDocument* document) {
+            m_helper.setFace(nullptr);
+            m_toolBox.disable();
```

Comment:
It'll be re-enabled the next time `UVView::selectionDidChange` is called and there are faces selected.

(I put the `m_toolBox.disable();` call there beacuse `UVView::documentWasCleared` also does that when there is no face selected)